README-idoverrideuser.md

```diff
@@ -0,0 +1,442 @@
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      sshpubkey:
+      - ssh-rsa AAAAB3NzaC1yc2EAAADAQABAAABgQCqmVDpEX5gnSjKuv97Ay ...
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview without
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview without
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      sshpubkey: []
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview with 1 certificate
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview with 1 certificate
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      certificate:
+      - "{{ lookup('file', 'cert1.b64', rstrip=False) }}"
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview with 3 certificate members
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview with 3 certificate members
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      certificate:
+      - "{{ lookup('file', 'cert1.b64', rstrip=False) }}"
+      - "{{ lookup('file', 'cert2.b64', rstrip=False) }}"
+      - "{{ lookup('file', 'cert3.b64', rstrip=False) }}"
+      action: member
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview without 2 certificate members
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview without 2 certificate members
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      certificate:
+      - "{{ lookup('file', 'cert2.b64', rstrip=False) }}"
+      - "{{ lookup('file', 'cert3.b64', rstrip=False) }}"
+      action: member
+      state: absent
+```
+
+
+Example playbook to make sure test user test_user is present in idview test_idview without certificates
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is present in idview test_idview without certificates
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      certificate: []
+```
+
+
+Example playbook to make sure test user test_user is absent in idview test_idview
+
+```yaml
+---
+- name: Playbook to make sure test user test_user is absent in idview test_idview
+  hosts: ipaserver
+  become: false
+
+  tasks:
+  - ipaidoverrideuser:
+      idview: test_idview
+      anchor: test_user
+      continue: true
+      state: absent
+```
+
```

Comment:
It would be nice to have an example playbook for "fallback_to_ldap".

Reply:
Fixed.